src/share/poudriere/jail.sh

```diff
@@ -70,6 +70,10 @@ Options:
                      Only applies if TARGET_ARCH and HOST_ARCH are different.
                      Will only be used if -m is svn*.
 
+Options for -d:
+    -C clean      -- Clean remaining data existing in pourdiere data folder.
+                     See poudriere(8) for more details. Can be one of:
+                       all, cache, logs, packages, wrkdirs
```

Comment:
Something seems off with whitespace here, can you ensure there's no tabs and only spaces for the usage area?